shesha-reactjs/src/components/panel/index.tsx

```diff
@@ -4,6 +4,7 @@ import { CollapseProps } from 'antd/lib/collapse';
 import classNames from 'classnames';
 import styled from 'styled-components';
 import { useStyles } from './styles/styles';
+import { isHidden } from '@/designer-components/_settings/components/utils';
```

Comment:
Any component on src/components level should not have dependencies on Designer or Settings (dependencies on src/designer-components folder)
Move this dependencies in CollapsiblePanelComponent